backend/dataall/db/connection.py

```diff
@@ -124,10 +124,12 @@ def get_engine(envname=ENVNAME):
         creds = json.loads(db_credentials_string['SecretString'])
         user = creds['username']
         pwd = creds['password']
+        host = param_store.get_parameter(env=envname, path='aurora/hostname')
+        database = param_store.get_parameter(env=envname, path='aurora/db')
+
         db_params = {
-            'host': param_store.get_parameter(env=envname, path='aurora/hostname'),
-            'port': param_store.get_parameter(env=envname, path='aurora/port'),
-            'db': param_store.get_parameter(env=envname, path='aurora/db'),
+            'host': host,
+            'db': database,
```

Comment:
Hello @nikpodsh I see that you removed the port. Do we need it to make a connection or was it unnecessary?

Reply:
Yeah, I double checked and it turned out that the port was used. Thanks for bringing this up!
We don't use the port while creating the database connection in the backend. So it should always roll back to the default value 5432. 
I can re-add it and this time make the port be configurable on the backend or delete the usages of the port (two in the local env and one `quicksight#create_analysis` ). What do you think would be better ?

Reply:
I don't think adding the port as a configuration adds any value for the user, just more "work" for them. When you say deleting the port you mean hardcoding it in local and quicksight?

Reply:
Yes, just hardcoding. By deleting I meant the deleting a configurable parameter (which is configurable only in a few places)

Reply:
Looks good